models/articles.models.js

```diff
@@ -50,3 +51,18 @@ exports.getCommentsByArticle = (articleId) => {
     return result.rows;
   });
 };
+
+exports.patchVotes = (articleId, votes) => {
+  return getVotes(articleId)
```

Comment:
think about what the purpose of this function is, is it to check if the article exists and/or to find the current votes value? 

Checking the article exists could be acheived through a more generic function and the db.query has access to to the current votes so these can both be achevied in other ways.

It is up to you if you want to make modifications based on this.